composer.json

```diff
@@ -18,8 +18,8 @@
     "behat/behat": "~3.1.0-rc2",
     "behat/mink-extension": "~2.0",
     "drupal/drupal-driver": "dev-master",
-    "symfony/dependency-injection": "~2.7",
-    "symfony/event-dispatcher": "~2.7"
+    "symfony/dependency-injection": "2.8.*",
+    "symfony/event-dispatcher": "2.8.*"
```

Comment:
@acbramley yeah, as long as the versions you pin are compatible with Drupal core's versions that should work.